### PR TITLE
Describe address parameter in pricing API

### PR DESCRIPTION
**Pull request by maarcingebala**
Document the recently added `address` param for pricing fields.

### CODE REVIEW

**Comment by patrys** [comment]
Does the address also affect availability?

**Comment by maarcingebala** [comment]
@patrys Yes and I'll document that as well. I originally wanted to do that in a separate PR but I can do it here. It would go in the same section anyway ("Querying Product Information").

**Review by krzysztofwolski — APPROVED** [review]

File: docs/developer/products.mdx [hunk 1]
```diff
@@ -337,6 +337,64 @@ type Money {
 }
 ```
 
+See [Prices](api-conventions/prices.mdx) for more information about money types.
```
Comment (karolkielecki) [inline docs/developer/products.mdx:340]: See [Prices](api-conventions/prices.mdx) for more information about the money types.

Comment (karolkielecki) [inline docs/developer/products.mdx:340]:
```suggestion
See [Prices](api-conventions/prices.mdx) for more information about the money types.
```

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**